pkg/kubernetes/podhelper/container_logs.go

```diff
@@ -39,13 +44,20 @@ func HasNoBadLogs(client kubernetes.Interface, pod *corev1.Pod, containerName st
 	re := regexp.MustCompile("(?i)(fatal)|(error)|(warn)|(fail)")
 	scanner := bufio.NewScanner(podLogsReader)
 	var badLogLines string
+	containsLogs := false
```

Comment:
I'd rename this for clarity. `containsLogs` tells me that it perhaps contains container logs. But I see that this is a boolean. What question does this yes/now variable answer?  Is